scala/scala.bzl

```diff
@@ -782,10 +782,9 @@ _common_attrs_for_plugin_bootstrapping = {
 }
 
 _common_attrs = _common_attrs_for_plugin_bootstrapping + {
-  # dependency_analyzer_mode_soon_to_be_removed will be replaced by using command line flag called 'strict_java_deps' (https://github.com/bazelbuild/bazel/issues/3295)
+  # using stricts scala deps is done by using command line flag called 'strict_java_deps' 
```

Comment:
> using stricts scala deps is done by using command line flag called 'strict_java_deps' 
switching mode to "on" means...

I don't think this is clear enough, I think language should reflect how to use the command line flag to turn the feature on